flipperdevices/flipperzero-good-faps
# [FL-3593] Picopass rework with new NFC API

# What's new

- Rework Picopass App with new NFC API

## Merge after https://github.com/flipperdevices/flipperzero-firmware/pull/3050 is merged

# Verification 

- Build SDK from https://github.com/flipperdevices/flipperzero-firmware/pull/3050 : `./fbt updater_package`
- Update local path to SDK : `ufbt update -t f7 -l <local-sdk-path>`
- Build and run Picopass App: `ufbt launch`
- Verify app is working as before

# Checklist (For Reviewer)
